pkg/controller/restore-operator/operator.go

```diff
@@ -45,10 +45,10 @@ type Restore struct {
 	etcdCRCli  versioned.Interface
 	kubeExtCli apiextensionsclient.Interface
 
-	// restoreCRs is a map of cluster name to restore cr.
+	// restoreCRs is a map of restore name to restore cr.
```

Comment:
After a thorough digging, these two maps are outdated if we don't use clusterName but use restoreName.

All we need is an index that provide Get(restore_name) -> CR.
- We don't need `restoreCRs` and `clusterNames `, and we can get rid those.
- We don't need `restoreNames` to be added. Instead, we can make use of indexer.Store to get the object.

Reply:
See https://github.com/kubernetes/client-go/blob/master/tools/cache/store.go#L194